extensions/localPublish/package.json

```diff
@@ -12,12 +12,14 @@
   "dependencies": {
     "@bfc/extension": "file:../../Composer/packages/extension",
     "@botframework-composer/types": "latest",
+    "@types/tcp-port-used": "1.0.0",
```

Comment:
Is this a production dependency?

Reply:
> Is this a production dependency?

yes, this is defined here: https://www.npmjs.com/package/@types/tcp-port-used

Reply:
@zidaneymar it looks to be exporting the types alone. Why can this be defined as a `devDependency` ?

Reply:
> @zidaneymar it looks to be exporting the types alone. Why can't this be defined as a `devDependency` ?

sorry I will fix this